include/device_config.h

```diff
@@ -28,17 +28,21 @@
 // #define HAS_BALANCE_BOARD_INTEGRATION
 
 // Enable the yukkuri voice talking clock
+#ifndef WITHOUT_AQUESTALK
```

Comment:
The aquestalk binary detection script sets a flag `AQUESTALK_FOUND`, so it may be a good idea to wrap this into that instead of a separate flag. 

In such case also change the error down below in the file to a warning to warn the user if the library wasn't found (and move this feature flag down there I guess)

Creating a feature flag for a feature flag as done here is a pattern I would really be against of.

Reply:
I did not get how to use that script? For me the project just fails to build until I comment `AQUESTALK` in `device_config.h`. And it annoys git of a changed file every time I switch the branches.

Reply:
The error you're getting is most likely not a linker failure but a sanity check coming from here: https://github.com/vladkorotnev/plasma-clock/blob/e03bad791247aadf27ce15e6f962c4a63d0a1040/include/device_config.h#L86 that fails when this script doesn't find the library and thus doesn't set the AQUESTALK_FOUND flag: https://github.com/vladkorotnev/plasma-clock/blob/develop/helper/aquestalk-detect.py

This originally was done to prevent enabling the AquesTalk feature flag when the library file is missing. But now that I think of it, it makes sense to just automatically enable the HAS_AQUESTALK flag when the library is present and disable it otherwise, hence the suggestion

Reply:
should this work?
```
#ifdef LIBAQUESTALK_FOUND
#define HAS_AQUESTALK
#endif
```

Reply:
I think so, and add a #warning in #else for good measure, that should fix it for good.

Reply:
Logging is never enough, especially in embedded and CI both of which we have :P